tuf/client/client.go

```diff
@@ -54,7 +54,7 @@ func (c *Client) Update() error {
 	if err != nil {
 		logrus.Debug("Error occurred. Root will be downloaded and another update attempted")
 		if err := c.downloadRoot(); err != nil {
-			logrus.Error("client Update (Root):", err)
+			logrus.Error("Client Update (Root):", err)
```

Comment:
heh, thanks.  :D  it was annoying me to see the upper vs lower cases appear in the tests :)